### ISSUE TITLE
ELI5 how to install argon on OpenWrt official 19.07.4?

### ISSUE THREAD

**Issue by Uka-Uka** [jerrykuku/luci-theme-argon#132]
Hi, Thanks for this awesome theme.

I'm new to OpenWrt world, and I really like your theme, not sure if it's possible to install it on 19.07.4 stable or not, If so may you please guide me on how to get it installed?

**Comment by jerrykuku** [comment]
First goto System->Software   Filter:Luci-base check your luci-base version, if version is git-20.252.69040, then you can download the v2.2.4 luci-theme-argon ipk form here: https://github.com/jerrykuku/luci-theme-argon/releases/download/v2.2.4/luci-theme-argon_2.2.4-20200821_all.ipk , and you can install the ipk with upload.

If not,  Plese click refresh button to opkg update  and install the lastest luci-base  in Availble packages. After that, do the first step.

**Comment by Uka-Uka** [comment]
Thank you, good sir! Your instructions were pretty clear. I've managed to install it and it's really a lovely theme. Keep up the good work.

I just want to point something out, the transparency isn't working on firefox, however, it's working flawlessly on chrome.

**Comment by jerrykuku** [comment]
On firefox you can look at this https://github.com/jerrykuku/luci-theme-argon/issues/119#issuecomment-687573812

**Comment by Uka-Uka** [comment]
Thanks! that fixed the problem, now it's working as it should.
